### PR TITLE
fix: icon background, footer offscreen, horizontal scrolling refracto…

### PR DESCRIPTION
**Pull request by DexinJ**
## Describe your changes
The ticket calls for fixing the profile page structure for all profile tabs. 

1. Changing page height to screen form a fixed height.
2. Changing the icon images so they have transparent background instead of grey
3. Added flex-col on a condition to prevent side-by-side behavior of sidebar and content when the screen becomes narrower.
4. Create a router inside the profile page to prevent all other profile pages from needing to redo what has been done here.
5. Fixed capitalization bug in tab-list.jsx

## Issue ticket number and link
[SLOP-10](https://tripleten-apiary.atlassian.net/jira/software/projects/SLOP/boards/2?selectedIssue=SLOP-10)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by skwisgaarr8-9** [comment]
> The icons in the sidebar currently have a gray background. This should be removed so that it matches the [mock](https://www.figma.com/file/NDxE2DhULzua1KRFG6d4hA/Slopopedia?type=design&node-id=242%3A10324&mode=dev)
- This task is complete. 

> The footer of the page is off-screen and you have to scroll down to see it even though there’s plenty of room to display it
- This task is complete.

> The responsiveness isn’t quite right; the sidebar menu adapts so that it shows a horizontal menu when the screen gets smaller, but the page content doesn’t show up beneath it. Currently it shows side-by-side and creates horizontal scrolling. Improve this experience so that the elements are more responsive.
- This task is complete however, at a screen width of 574px, the horizontal menu has horizontal scrolling. This should be improved in someway e.g. rearrange the buttons, create a hamburger menu (probably not ideal as the header is already a hamburger menu), etc. 

> Ensure these changes are present on all of the profile pages-- the layout needs to be consistent for each menu tab.
- This task is complete

**Comment by DexinJ** [comment]
> > The responsiveness isn’t quite right; the sidebar menu adapts so that it shows a horizontal menu when the screen gets smaller, but the page content doesn’t show up beneath it. Currently it shows side-by-side and creates horizontal scrolling. Improve this experience so that the elements are more responsive.
> 
> * This task is complete however, at a screen width of 574px, the horizontal menu has horizontal scrolling. This should be improved in someway e.g. rearrange the buttons, create a hamburger menu (probably not ideal as the header is already a hamburger menu), etc.

I checked figma and the horizontal scroll seems to be by design

File: src/routes/profile-route/profile-horizontal-menu.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const ProfileHorizontalMenu = () => {
     {
       title: "Me goblin",
       src: sidebarCrown,
-      link: "/profile/me",
+      link: "/profile",
```
Comment (SamPage4427) [inline src/routes/profile-route/profile-horizontal-menu.jsx:12]: Might look better if the horizontal menu items are centered?
image
The extra space at the end looks off. Should probably have the same margins on each side so it centers better on the screen.

File: src/routes/profile-route/profile-route.jsx [hunk 2]
```diff
@@ -9,20 +11,22 @@ export const ProfileRoute = () => {
   });
 
   return (
-    <div className="max-w-[1440px] min-h-[1023px] bg-gray-background mx-auto">
+    <div className="max-w-[1440px] bg-gray-background mx-auto min-h-screen relative">
```
Comment (SamPage4427) [inline src/routes/profile-route/profile-route.jsx:14]: The header is currently changing to be larger than the max width of 1440px. I'd suggest removing it from this for now.

**Review by SamPage4427 — COMMENTED** [review]
Everything looks good so far. There are a couple small issues. Other than that I'm not seeing any glaring issues with this ticket other than we'll need a ticket for the profile pages responsiveness for the content of each page. Nice work so far.

**Review by jahorwitz — APPROVED** [review]

**Pull request merged**